.gitignore

```diff
@@ -30,3 +30,9 @@ GPATH
 .submodule
 *.built
 *.completed
+
+/esp-idf*
+/esp-quick-toolchain/
+/esp32/
+/rp2040/
+/samples/Basic_Serial/files/
```

Comment:
If it bothers you. But this should be in `/samples/Basic_Serial/.gitignore`.

Reply:
Moved